Gemfile

```diff
@@ -3,6 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.3'
 
+gem 'devise'
```

Comment:
Guys it is a good practice to put a fixed version to the gems, it is common to forget that (happens to me too) but in the real world, you will be in trouble if a gem is updated to a recent version and it breaks your code. This is especially true for strange gems you need on projects from time to time.